### ISSUE TITLE
Location of resources folder

### ISSUE THREAD

**Issue by dhirschfeld** [mingrammer/diagrams#163]
At present it appears that the `resources` folder is distributed under `site-packages/resources` rather than `site-packages/diagrams/resources`.

It seems a bit strange to deploy non-python package data directly to the `site-packages` folder.

Would there be any objection to a PR which moved the `resources` folder to  `site-packages/diagrams/resources`?

**Comment by mingrammer** [comment]
It makes sense. It would be nice to make the `diagrams` folder the project root folder. I'll work on it. Thank you for pointing it.

**Comment by dhirschfeld** [comment]
I'd be happy to work on it but I don't know anything about `poetry`.

I'd imagine simply putting the `resources` folder directly under `diagrams` would do the job but if you wanted to keep the existing source structure it might need some config to put it in the right place.

**Comment by mingrammer** [comment]
Yes, just putting the `resources` folder under `diagrams`, and update the icon loading method for nodes classes.
